_aggregations/bucket/composite.md

```diff
@@ -0,0 +1,826 @@
+| `time_zone`         | Optional           | String                | A string representing the time offset from UTC, either as an [ISO 8601 UTC offset](https://en.wikipedia.org/wiki/UTC_offset) ("-07:00") or as a TZ timezone database [identifier](https://en.wikipedia.org/wiki/Tz_database) ("America/Los_Angeles").|
+
+### Example: Date interval
+
+The following example request returns the first `4` composite buckets for the year each sold product was created and the date it was sold in the OpenSearch Dashboards sample e-commerce data, based on bucket widths of one year and one day respectively:
+
+```json
+GET opensearch_dashboards_sample_data_ecommerce/_search
+{
+  "size": 0,
+  "aggs": {
+    "composite_buckets": {
+      "composite": {
+        "sources": [
+          { "product_creation_date": { "date_histogram": { "field": "products.created_on", "calendar_interval": "1y", "format": "yyyy" }}},
+          { "order_date": { "date_histogram": { "field": "order_date", "calendar_interval": "1d", "format": "yyyy-MM-dd" }}}
+        ],
+        "size": 4
+      }
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+### Example response: Date interval
+
+As shown in the following example response, the aggregation returns the formatted date-based bucket keys and counts. For `date_interval` composite aggregation, field ordering is by date:
+
+```json
+{
+  "took": 21,
+  "timed_out": false,
+  "_shards": {
+    "total": 1,
+    "successful": 1,
+    "skipped": 0,
+    "failed": 0
+  },
+  "hits": {
+    "total": {
+      "value": 4675,
+      "relation": "eq"
+    },
+    "max_score": null,
+    "hits": []
+  },
+  "aggregations": {
+    "composite_buckets": {
+      "after_key": {
+        "product_creation_date": "2016",
+        "order_date": "2025-02-23"
+      },
+      "buckets": [
+        {
+          "key": {
+            "product_creation_date": "2016",
+            "order_date": "2025-02-20"
+          },
+          "doc_count": 146
+        },
+        {
+          "key": {
+            "product_creation_date": "2016",
+            "order_date": "2025-02-21"
+          },
+          "doc_count": 153
+        },
+        {
+          "key": {
+            "product_creation_date": "2016",
+            "order_date": "2025-02-22"
+          },
+          "doc_count": 143
+        },
+        {
+          "key": {
+            "product_creation_date": "2016",
+            "order_date": "2025-02-23"
+          },
+          "doc_count": 140
+        }
+      ]
+    }
+  }
+}
+```
+
+## Geotile grid
+
+Use `geotile_grid` sources to aggregate `geo_point` values into buckets representing map tiles. As with the other composite aggregation `sources`, by default results include only buckets that contain data.
+
+Each cell corresponds to a [map tile](https://en.wikipedia.org/wiki/Tiled_web_map). Cell labels use a "{zoom}/{x}/{y}" format.
```

Comment:
Is there a site that does a good job of explaining the x,y system that geo-grid uses? 

Does OpenSearch use one of the available geo-grid standards?